docs/manual/troubleshooting.tex

```diff
@@ -947,18 +947,32 @@ \subsection{Configure Eclipse to edit the Checker Framework\label{building-eclip
 \begin{enumerate}
 \item Follow the instructions earlier in Section~\ref{build-source} to
 clone and build all projects from their sources.
+\item Follow the instructions in the \<README-eclipse> document found in the
+\<\$CHECKERFRAMEWORK/../annotation-tools/annotation-file-utilities/lib>
+directory to download JDK 8 source code and create symbolic links to
+\<tools.jar> and \<src.zip>.
 \item Download Eclipse from
 the \href{https://www.eclipse.org/downloads/}{official Eclipse website}
 and install it.
-\item Run Eclipse.
-\item Enter the main Eclipse working screen and in the ``File'' menu,
-select ``Import'' $\rightarrow$ ``General'' $\rightarrow$ ``Existing
-Projects into Workspace''.
+\item Run Eclipse and enter the main Eclipse working screen.
+\item Checker Framework's sources depend on classes packaged in \<rt.jar> from
```

Comment:
Is this really for rt.jar? If a user doesn't do anything, Eclipse wouldn't even show me javadoc for `java.util.List`?

Reply:
It would show javadoc for `java.util.List` without the source attachment, but it won't show the source code for that class.